docker-swarm-cluster/azuredeploy-parameters.json

```diff
@@ -0,0 +1,68 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
```

Comment:
Changed! 

By the way, the template example uses a azuredeploy.parameters.json file but the bot tries to find an azuredeploy-parameters.json file. 

I don't know if it's a bug on the bot or in the example.